tests/connect/test_analyze_plan.py

```diff
@@ -0,0 +1,11 @@
+from __future__ import annotations
+
+
+def test_analyze_plan(spark_session):
```

Comment:
so i think we should change this to the commented out assertion, then do something like

```suggestion
@pytest.mark.skip(
    reason="Currently an issue in the spark connect code. It always passes the inferred schema instead of the supplied schema."
)
def test_analyze_plan(spark_session):

```